### PR TITLE
refac: Remove market role dataframe

### PR DESCRIPTION
**Pull request by JesperGomes**
## Description

market role is not needed since we its content is part of metering point periods

## References

* #0000

### CODE REVIEW

File: source/databricks/calculation-engine/package/schemas/metering_point_schema.py
Comment (BjarkeMeier) [inline source/databricks/calculation-engine/package/schemas/metering_point_schema.py:1]: What is the purpose of this schema? We already have the `metering_point_period_schema`.

File: source/databricks/calculation-engine/package/steps/wholesale/tariff_calculators.py [hunk 1]
```diff
@@ -11,28 +11,38 @@
...
 from package.constants import Colname
 
-total_quantity = "total_quantity"
-charge_count = "charge_count"
 
-
-def calculate_tariff_price_per_ga_co_es(
-    spark: SparkSession, tariffs: DataFrame
-) -> DataFrame:
+def calculate_tariff_price_per_ga_co_es(tariffs: DataFrame) -> DataFrame:
```
Comment (BjarkeMeier) [inline source/databricks/calculation-engine/package/steps/wholesale/tariff_calculators.py:19]: What is "co" short for?

**Review by BjarkeMeier — APPROVED** [review]
Not easy stuff to review. But to me, it looks like great work 👍

File: source/databricks/calculation-engine/package/steps/wholesale/tariff_calculators.py (hunk 1 above)
Comment (JesperGomes) [reply source/databricks/calculation-engine/package/steps/wholesale/tariff_calculators.py:19]: Charge_owner

File: source/databricks/calculation-engine/package/schemas/metering_point_schema.py
Comment (JesperGomes) [reply source/databricks/calculation-engine/package/schemas/metering_point_schema.py:1]: Yes, I also think we can remove it (might require a few adjustments in the tests that use it)

**Pull request merged**